module-bsp/board/rt1051/os/fsl_tickless_gpt.c

```diff
@@ -11,8 +11,9 @@
 #include "task.h"
 #include "fsl_tickless_generic.h"
 
-#if configUSE_TICKLESS_IDLE == 1
+#if configUSE_TICKLESS_IDLE == 2
```

Comment:
why change from 1 to 2? is it changed everywhere?

Reply:
It is described here: https://www.freertos.org/low-power-ARM-cortex-rtos.html 
Basically, set it to 2 if you want to use custom `vPortSuppressTicksAndSleep` and `vPortSetupTimerInterrupt`. It could be simply overwritten as these symbols are defined weak. Unfortunately, there is no way to overwrite weak symbols defined in different translation unit.